### PR TITLE
Unified close icons

### PR DESCRIPTION
**Pull request by pajasevi**
I changed all close icons to FaClose (look way better than MdClose) except for GlobalError component which will be rewritten anyway.

I also added close icon to PeerForm component.

Second commit is just code cleaning.

Solves #145

### CODE REVIEW

File: app/components/ChannelForm/ChannelForm.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ const ChannelForm = ({
       className={styles.modal}
     >
       <div onClick={closeChannelForm} className={styles.modalClose}>
-        <FaClose className={styles.close} />
```
Comment (Empact) [inline app/components/ChannelForm/ChannelForm.js]: Looks like styles.close is unused after this removal. You can drop it from the scss as well.

Comment (pajasevi) [reply app/components/ChannelForm/ChannelForm.js]: Right! Thanks for the tip, Webstorm didn't show me that.

Comment (pajasevi) [reply app/components/ChannelForm/ChannelForm.js]: It's not in the SCSS at all, that's why I removed it https://github.com/pajasevi/zap-desktop/blob/95872d8ea3ed76f50ece53173e0871f2f83269b6/app/components/ChannelForm/ChannelForm.scss

**Comment by JimmyMow** [comment]
Nice @pajasevi, looks good to me. Merging

**Pull request merged**